sdk/python/tests/test_client.py

```diff
@@ -324,83 +298,138 @@ def test_version(self, mocked_client, mocker):
         ],
     )
     def test_get_online_features(self, mocked_client, auth_metadata, mocker):
-        ROW_COUNT = 300
+        assert 1 == 1
 
-        mocked_client._serving_service_stub = Serving.ServingServiceStub(
+    @pytest.mark.parametrize(
+        "mocked_client",
+        [
+            lazy_fixture("mock_client"),
+            lazy_fixture("mock_client_with_auth"),
+            lazy_fixture("secure_mock_client"),
+            lazy_fixture("secure_mock_client_with_auth"),
+        ],
+    )
+    def test_get_historical_features(self, mocked_client, mocker):
+        assert 1 == 1
+
+    @pytest.mark.parametrize(
+        "mocked_client",
+        [lazy_fixture("mock_client"), lazy_fixture("secure_mock_client")],
+    )
+    def test_get_entity(self, mocked_client, mocker):
+        mocked_client._core_service_stub = Core.CoreServiceStub(
             grpc.insecure_channel("")
         )
 
-        def int_val(x):
-            return ValueProto.Value(int64_val=x)
+        entity_proto = EntityProto(
+            spec=EntitySpecProto(
+                name="driver_car_id",
+                description="Car driver id",
+                value_type=ValueProto.ValueType.STRING,
+                labels={"key1": "val1", "key2": "val2"},
+            ),
+            meta=EntityMetaProto(),
+        )
 
-        request = GetOnlineFeaturesRequest(project="driver_project")
-        request.features.extend(
-            [
-                FeatureRefProto(feature_set="driver", name="age"),
-                FeatureRefProto(name="rating"),
-                FeatureRefProto(name="null_value"),
-            ]
+        mocker.patch.object(
+            mocked_client._core_service_stub,
+            "GetEntity",
+            return_value=GetEntityResponse(entity=entity_proto),
         )
-        recieve_response = GetOnlineFeaturesResponse()
-        entity_rows = []
-        for row_number in range(1, ROW_COUNT + 1):
-            request.entity_rows.append(
-                GetOnlineFeaturesRequest.EntityRow(
-                    fields={"driver_id": int_val(row_number)}
-                )
+        mocked_client.set_project("my_project")
+        entity = mocked_client.get_entity("my_entity")
```

Comment:
What are we testing here? All these tests seem like they set the response and then run the method that gets that response.

Reply:
Yup, its testing that the method returns the expected response type and value. Actually, most of these flow are already tested in the e2e tests. Previously we have all these unit tests and e2e tests, but they essentially test the same flow. Should we just keep e2e tests in these cases?

Reply:
Removed.